r-package/R/read_intermediate_region.R

```diff
@@ -1,68 +1,63 @@
-#' Download shape files of Brazil's Intermediate Geographic Areas as sf objects.

```

Comment:
What exactly has been changed here? Github indicates that you have replaced the whole function

Reply:
Curious. In all files, I removed the following `library(geobr)` in the examples. In `r-package/R/read_intermediate_region.R`, I also remove the period at the end of the function title. I run a CRAN check and got no errors.